sync/sync.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+import os
+import pkgutil
+
+from github import Github
+
+
+def run():
+    g = Github(os.getenv("GITHUB_TOKEN"))
+
+    # Generate dictionary of teams
+    teams = {}
+    for team in g.get_organization("pennlabs").get_teams():
+        if team.name == "Alumni":  # Found an Alumni Team
+            teams.setdefault("Alumni", []).append(team)
```

Comment:
Why are these capital? I thought dict keys were normally lowercase

Reply:
I just looked it up and couldn't find anything conclusive, but I agree with you. Will change to lowercase.